Formula/rocksdb.rb

```diff
@@ -53,6 +53,10 @@ def install
   end
 
   test do
+    # Work around Xcode 11 clang bug
+    # https://bitbucket.org/multicoreware/x265/issues/514/wrong-code-generated-on-macos-1015
+    ENV.append_to_cflags "-fno-stack-check" if DevelopmentTools.clang_build_version >= 1010
```

Comment:
This shouldn't be needed as of Xcode 11.4.1

Reply:
I thought this might be a solution for solving the compilation issue on the osx 10.15.